documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -13,26 +13,35 @@
 
 = Web Application Deployment
 
+The Jetty server supports the deployment of these types of web applications:
+
+* <<web-application-format-static,Jetty Static web applications>> -- web applications made of static files only.
+* <<web-application-format-core,Jetty Core web applications>> -- web applications written against the Jetty `Handler` APIs, without any Servlet dependencies.
+* <<web-application-format-jakarta,Java EE 8 web applications>> -- web applications written using the Servlet 4.0 (and associated specifications) in the `+javax.*+` packages.
+* <<web-application-format-jakarta,Jakarta EE 9 to Jakarta {ee-current-caps} web applications>> -- web applications written using the corresponding Servlet (and associated specifications) in the `+jakarta.*+` packages.
+
+In the following, the term "web application" is used to refer to either a Jetty Static web application, or Jetty Core web application, or Java EE 8 web application or Jakarta EE 9 to Jakarta {ee-current-caps} web application, unless specified otherwise.
```

Comment:
When coming up with the coreapp vs webapp vs staticapp naming convention, we explicitly said that we would not use "web application" in the generic sense as it overloads that name in a confusing way and if somebody says "deploy the web application" we have no idea what they mean.

Let's pick a term for the generic.   It could just be "application", but I still don't mind "context".  Whilst "context" is somewhat overloaded, we do know that every "application" has a `ContextHandler` that is configured with a `contextPath` and that requests entering the "application" are given a `Context` instance that describes the "application" and that for web applications that context is a `ServletContext`.  It is all context!

Reply:
I agree with @gregw . We specifically used the terms `coreapp`, `webapp` and `staticapp`. We cannot not now refer to all of them as web applications without nullifying the clear delineation of the types of application. So IMHO they should be referred to as a group as `applications`, which is a widely understood term, and fits nicely with the specialized cases of `core application`, `web application` and `static application`.